packages/google-types/src/constants/intents.ts

```diff
@@ -14,6 +14,9 @@ export enum GoogleStatusIntent {
   MEDIA_FINISH = 'actions.intent.MEDIA_STATUS_FINISHED',
   MEDIA_FAIL = 'actions.intent.MEDIA_STATUS_FAILED',
 }
+
+// check that intent is in the GoogleIntent enum, returning boolean
+export const isGoogleIntent = (intent: any): intent is GoogleIntent => Object.values(GoogleIntent).includes(intent);
```

Comment:
so what is happening here is you are creating an array out of `GoogleIntent`'s keys and running a `O(n)` check against `intent` (comparing all values).

Use hasOwnProperty: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty